backend/src/packages/business/business.controller.ts

```diff
@@ -368,6 +376,39 @@ class BusinessController extends Controller {
           }>,
         ),
     });
+
+    this.addRoute({
+      path: BusinessApiPath.TRUCKS,
+      method: 'GET',
+      authStrategy: defaultStrategies,
+      validation: {
+        params: businessGetAllTrucksParameters,
+      },
+      handler: (options) =>
+        this.findAllTrucks(
+          options as ApiHandlerOptions<{
+            params: BusinessGetAllTrucksRequestParameters;
+            query: PaginationPayload;
+          }>,
+        ),
+    });
+
+    this.addRoute({
+      path: BusinessApiPath.TRUCKS,
+      method: 'POST',
+      authStrategy: defaultStrategies,
```

Comment:
creating new truck can be done only by user with business group, is it working now?

Reply:
yes, defaultStrategies contains verification for a business group.